src/field_theory/krull_topology.lean

```diff
@@ -8,7 +8,7 @@ import field_theory.galois
 import topology.algebra.filter_basis
 import topology.algebra.open_subgroup
 import tactic.by_contra
-
+import topology.category.Profinite.default
```

Comment:
Why do you need this import? Can't you replace it with something lower in the hierarchy?